incremental/lib/functions.sh

```diff
@@ -240,6 +240,10 @@ function prepare_pike {
     cp /opt/rpc-upgrades/playbooks/patches/pike/run-upgrade.patch /opt/openstack-ansible
     patch -p1 < run-upgrade.patch
   popd
+
+  if [[ ! -f "/etc/openstack_deploy/queens_upgrade_prep.complete" ]]; then
```

Comment:
This is under the prepare_pike section, you'll probably want to move it under prepare_queens along with changing into the playbooks directory.

Reply:
👍Thanks, late night PRs snafu...